MonsterWaves/Screen/BaseScreen.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "Screen/Window.hpp"
+
+#include "memory"
+
+class BaseScreen
+{
+public:
+	BaseScreen(Window* window);
+	virtual ~BaseScreen() = default;
+public:
+	virtual void update(float dt) = 0;
+	virtual void render() = 0;
+	virtual void handleEvents() = 0;
```

Comment:
@JakubMastalski handleEvents nie powinna być czysto wirtualne. Spójrz jak wygląda to u mnie.
- [ ] umieścić bazową implementację tej metody (zdarzenia okna)